### PR TITLE
fix(list/toolbar): pass all sub props to components

### PR DESCRIPTION
**Pull request by jmfrancois**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?**

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)

The toolbar do not pass the sortDesc boolean props to the selectsortby component

**What is the new behavior?**

All subprops are passed.
tests added

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: src/List/Toolbar/Toolbar.component.js [hunk 1]
```diff
@@ -4,24 +4,36 @@ import SelectDisplayMode from './SelectDisplayMode';
 import SelectSortBy from './SelectSortBy';
 import Filter from './Filter';
 
+export function getSubProps(props, component) {
+	const subProps = {};
+	Object.keys(component.propTypes).forEach((key) => {
```
Comment (jsomsanith) [inline src/List/Toolbar/Toolbar.component.js]: Array.filter().map() ?

File: src/List/Toolbar/Toolbar.component.js [hunk 2]
```diff
@@ -4,24 +4,36 @@ import SelectDisplayMode from './SelectDisplayMode';
 import SelectSortBy from './SelectSortBy';
 import Filter from './Filter';
 
+export function getSubProps(props, component) {
```
Comment (acateland) [inline src/List/Toolbar/Toolbar.component.js]: you should turn this function into a component

File: src/List/Toolbar/Toolbar.component.js (hunk 1 above)
Comment (jmfrancois) [inline src/List/Toolbar/Toolbar.component.js]: map is to return an array, I don't need an array but an object.
I may have missed something in your comment

Comment (jsomsanith) [inline src/List/Toolbar/Toolbar.component.js]: Sorry, i'm just tired :(

**Pull request merged**